plugins/travis-ci/dev/index.tsx

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2020 Spotify AB
```

Comment:
Remove these copyright headers at the end please.

Reply:
But it violates the aslant rules

Reply:
Ok. Don't worry about it for the moment. I will fix it at the end.